src/Interface/Tabs/PluginList.as

```diff
@@ -159,7 +168,26 @@ class PluginListTab : Tab
 				UI::TableNextColumn();
 				Controls::PluginCard(m_plugins[i], colWidth);
 			}
+
+			float rowHeight = Math::Ceil(UI::GetScrollMaxY() / (m_plugins.Length / Setting_PluginsPerRow + 1));
+
+			// draw a message before we check for being near the end
+			bool haveMorePages = m_page + 1 < m_pageCount;
+			if (haveMorePages) {
+				UI::TableNextRow(UI::TableRowFlags::None, rowHeight);
+				UI::TableNextColumn();
+				string infiniteScrollMsg = (m_request is null ? "Scroll to Load" : "Loading") + " Page " + (m_page + 2);
```

Comment:
Make it lowercase `page` instead of `Page`

Reply:
I changed "Load" to "load", too. (Though that message is hard to see without furious scrolling)